corehq/apps/hqwebapp/management/commands/generate_webpack_settings.py

```diff
@@ -0,0 +1,22 @@
+import json
+from pathlib import Path
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+
+import corehq
+
+WEBPACK_BUILD_DIR = Path(corehq.__file__).resolve().parent.parent / "webpack" / "_build"
+
+
+class Command(BaseCommand):
+    help = ("Generates a settings file for webpack so that the same staticfiles directory "
```

Comment:
Is this needed because webpack needs an absolute path that will contain an environment directory? Like `/staging/` or a deploy directory?

Reply:
webpack needs to know where to put the bundles, so it will either use the default staticfiles root or one that's specified for the environment